### DISCUSSION TITLE
Geolocation

### DISCUSSION THREAD

**Discussion by srlamin19** [immich-app/immich#502] [Q&A]
Dear,

I'm not sure how to use **MAPBOX_KEY,** is someone using _**Geolocation**_ already ? 

Could you please share how to set ?

Thanks!

Version: v1.24.0
Using as Server RPI-4

**Comment by alextran1502** [comment]
1. Enable `ENABLE_MAPBOX=true`
2. Register an account from https://account.mapbox.com/
3. Get the default public token and put it in `MAPBOX_KEY=your-token`. Please restart the server to register the new added values

**Comment by srlamin19** [comment]
Okay, settings updated and server restarted.

So far, I can not see how it works.. Maybe takes more time as I expected .. I check my account on MAPBOX, is consuming requests on my free geocoding api..


![IMG_1533](https://user-images.githubusercontent.com/111645772/185783266-35da9750-851c-4bc1-80c5-714588008b83.PNG)
![IMG_1534](https://user-images.githubusercontent.com/111645772/185783388-f546fb3c-b28c-415b-82be-bbf321a2d9bb.PNG)
